ui/src/shared/constants/colors.js

```diff
@@ -5,7 +5,7 @@
 * there. Lets try and keep different color variations down ot a minimum.
 * */
 
-const base = {
+let base = {
```

Comment:
Why don't you move the `base` declaration some lines below, so that it doesn't need to be `let`?